front/src/applications/operationalStudies/views/Scenario.tsx

```diff
@@ -11,7 +11,8 @@ import TimetableManageTrainSchedule from 'modules/trainschedule/components/Timet
 import BreadCrumbs from 'applications/operationalStudies/components/BreadCrumbs';
 import { getInfraID, getTimetableID } from 'reducers/osrdconf/selectors';
 import { useModal } from 'common/BootstrapSNCF/ModalSNCF';
-import { FaEye, FaEyeSlash, FaPencilAlt } from 'react-icons/fa';
+import { GoPencil } from 'react-icons/go';
+import { FaEye, FaEyeSlash } from 'react-icons/fa';
```

Comment:
```suggestion
import { GoEye, GoEyeClosed, GoPencil, GoChevronLeft, GoChevronRight } from 'react-icons/go';
```

GoEye and GoEyeClosed can be used at **line 239**

GoChevronLeft can replace` i className="icons-arrow-prev"` at **line 246**

GoChevronRight can replace` i className="icons-arrow-next"` at **line 333**